modules/doc/config.yml

```diff
@@ -37,9 +37,31 @@ Extensions:
         repo: https://github.com/idaholab/moose
         google-cse: 000781241768298771085:pedtnlxdkx8
         menu:
-            Getting Started: getting_started.menu.md
-            Documentation: documentation.menu.md
-            Help: help.menu.md
+            Getting Started:
+                Installing MOOSE: getting_started/installation/index.md
+                Creating an Application: getting_started/new_users.md
+                Tutorials: getting_started/examples_and_tutorials/index.md
+                Learning Resources: getting_started/learning_resources/index.md
+            Documentation:
+                Application Usage: application_usage/index.md
+                Application Development: application_development/index.md
+                Framework Development: framework_development/index.md
+                Modules: modules/index.md
+                INL HPC: help/inl/index.md
+                MOOSEDocs: MooseDocs/index.md
+                Infrastructure: infrastructure/index.md
+                Syntax Index: syntax/index.md
+                Source Index: /source/index.md
+                A-to-Z Index: help/a-to-z.md
+            Help:
+                Frequently Asked Questions: help/faq/index.md
+                Help Forum: https://github.com/idaholab/moose/discussions
```

Comment:
should we add a debugging page here?

Reply:
Just link to this page?: https://mooseframework.inl.gov/docs/PRs/22237/site/application_usage/failed_solves.html. Or is there more to debugging? Or should we just do as the the first FAQ entry "why does my solve fail?"?

Reply:
Yeah there is too much to debugging. 
For devs there is this one
https://mooseframework.inl.gov/application_development/debugging.html
For users
that other one is not bad but pretty incomplete
https://mooseframework.inl.gov/application_usage/failed_solves.html